integration_test/gce/gce_testing.go

```diff
@@ -739,6 +739,31 @@ func runCommand(ctx context.Context, logger *log.Logger, stdin io.Reader, args [
 	return output, err
 }
 
+// getGcloudConfigDir returns the current gcloud configuration directory.
+func getGcloudConfigDir(ctx context.Context) (string, error) {
+	out, err := RunGcloud(ctx, log.New(io.Discard, "", 0), "", []string{"info", "--format=value[terminator=''](config.paths.global_config_dir)"})
+	if err != nil {
+		return "", err
+	}
+	return out.Stdout, nil
+}
+
+// SetupGcloudConfigDir sets up a new gcloud configuration directory.
+// This copies the "configurations" subdirectory of the context-specified
```

Comment:
this comment is stale, it copies the whole thing.

Reply:
Done.